ddtrace/contrib/aiobotocore/__init__.py

```diff
@@ -32,7 +32,11 @@
 
 with require_modules(required_modules) as missing_modules:
     if not missing_modules:
-        from .patch import get_version
-        from .patch import patch
+        # Required to allow users to import from `ddtrace.contrib.aiohttp.patch` directly
+        from . import patch as _  # noqa: F401, I001
```

Comment:
Without this line `ddtrace.contrib.aiobotocore import patch` ALWAYS imports the `ddtrace.contrib.aiobotocore.patch.py` module and not the `ddtrace.contrib.aiobotocore.patch(...)` function. 

This line is required to avoid a breaking change.